packages/manager/apps/office/src/index.js

```diff
@@ -0,0 +1,19 @@
+import 'regenerator-runtime/runtime';
+import 'script-loader!jquery'; // eslint-disable-line
+import 'script-loader!moment/min/moment.min.js';// eslint-disable-line
+import 'whatwg-fetch';
+import { attach as attachPreloader } from '@ovh-ux/manager-preloader';
+import { bootstrapApplication } from '@ovh-ux/manager-core';
+
+attachPreloader();
+
+bootstrapApplication().then(({ region }) => {
+  import(`./config-${region}`)
+    .catch(() => {})
+    .then(() => import('./app.module'))
+    .then(({ default: application }) => {
+      angular.bootstrap(document.body, [application], {
+        strictDi: false,
```

Comment:
true?

Reply:
I'll try but I'm not sure it the office module is strict di proof